game_sim.py

```diff
@@ -866,6 +866,8 @@ async def compare_stats(length):
                         percent_diff = 1 - (predicted / actual)
                         hitting_diffs["below"][pid][key] = percent_diff
                     else:
+                        if predicted == 0:
```

Comment:
did you actually encounter an error here? 
this can only occur if actual == predicted == 0 and we may want to handle the output differently in those situations.

Reply:
I saw this when I set iterations to 1